source/reference/service-limitations.txt

```diff
@@ -189,6 +190,26 @@ Reading:
    When you intend to have many concurrent writers, have each writer
    work on a separate document. 
 
+.. _mongodb-service-limitations-sync-default-message-size-limit:
+
+Default Message Size Limit
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Device Sync has a default message size limit of {+sync-default-message-size-limit+}
+compressed for any messages sent from the client application. The message size
+depends on the compressed size of changes sent from the client device. This is 
+not directly equivalent to the size of any document in the upload. In practice,
+however, the changeset size is generally on the same order of magnitude of
+the size of documents in the upload.
+
+This limit acts as a countermeasure against connections hogging too many 
+resources.
```

Comment:
> ...hogging too many resources.

We shouldn't use a colloquial term like "hogging" for a couple of reasons:

- Translation. We need to keep in mind that the docs are now being translated into multiple languages. Plain English is better for this, partly due to the next point.
- Specificity. It's unclear what "hogging" means in this category. Using up compute, taking up bandwidth, etc.

Any chance we can be more specific with this sentence?